pennylane_lightning/src/simulators/lightning_qubit/gates/CMakeLists.txt

```diff
@@ -15,48 +15,48 @@ if (ENABLE_GATE_DISPATCHER AND UNIX AND (${CMAKE_SYSTEM_PROCESSOR} MATCHES "(AMD
                             AssignKernelMap_AVX2.cpp
                             AssignKernelMap_AVX512.cpp
                             AssignKernelMap_Default.cpp CACHE INTERNAL "" FORCE)
-    add_library(lightning_qubit_gates_kernel_map STATIC ${KERNEL_MAP_FILES})
-    target_link_libraries(lightning_qubit_gates_kernel_map PRIVATE  lightning_qubit_utils lightning_external_libs lightning_compile_options)
-    set_target_properties(lightning_qubit_gates_kernel_map PROPERTIES POSITION_INDEPENDENT_CODE ON)
+    add_library(lqg_kernel_map STATIC ${KERNEL_MAP_FILES})
```

Comment:
Thanks @AmintorDusko, good to know, maybe that will show up in LKokkos too.

Reply:
I hope not, but if it appears you are ready! :muscle: